source/Structs.h

```diff
@@ -441,9 +443,9 @@ namespace MDLStructs
 		int32_t skinRefCount;
 		int32_t skinFamilyCount;
 		int32_t skinRefOffset;
-		inline int16_t GetSkin(const int family, const int material) const
+		[[nodiscard]] inline int16_t GetSkin(const int family, const int material) const
 		{
-			const int16_t* skinTbl = reinterpret_cast<const int16_t*>(reinterpret_cast<const uint8_t*>(this) + skinRefOffset);
+			const auto* skinTbl = reinterpret_cast<const int16_t*>(reinterpret_cast<const uint8_t*>(this) + skinRefOffset);
```

Comment:
Was there a reason for changing this to an inferred type?

Reply:
CLion likes auto, I'll revert it

Reply:
Probably the static cast making it trivially inferable, but given we still need to write out `auto` I generally prefer to keep it more explicit.

If this is something failing clang-tidy and not CLion’s own warnings then I’m happy to keep this in though

Reply:
Nah it was just CLion, I rebased onto master and reverted it